react/src/lib/EditableTextfield/index.js

```diff
@@ -66,10 +66,13 @@ class EditableTextfield extends React.Component {
   }
 
   handleEsc = e => {
+    var value = e.target.value;
+    var handleDoneEditing = this.props.handleDoneEditing;
     this.setState(
       {
         isEditing: false
-      }
+      },
+      () => handleDoneEditing && handleDoneEditing(e, {value})
```

Comment:
pressing escape while editing is the same as pressing enter without editing, so just like in the case of "enter" key being pressed, escape should also return to the `handleDoneEditing` method if available. Also helps in putting the focus back on this component after editing is done because currently focus gets lost